arlo-client/src/components/Audit/useBallotManifest.ts

```diff
@@ -59,11 +78,23 @@ const useBallotManifest = (
     return false
   }
 
+  const deleteBallotManifest = async (): Promise<boolean> => {
+    if (await deleteBallotManifestFile(electionId, jurisdictionId)) {
+      setBallotManifest(await loadBallotManifest(electionId, jurisdictionId))
+      return true
+    }
+    return false
+  }
+
   useEffect(() => {
     const isFinishedProcessing = (manifest: IFileInfo) =>
       !!(manifest.processing && manifest.processing.completedAt)
 
-    if (!ballotManifest || isFinishedProcessing(ballotManifest)) return
+    if (
+      !(ballotManifest && ballotManifest.file) ||
+      isFinishedProcessing(ballotManifest)
+    )
+      return
```

Comment:
This change fixes a bug where we were initiating polling after loading the initial ballot manifest state from the backend when there was no ballot manifest uploaded. That state looks like `{file: null, processing: null}`